h1b/src/usb/registers.rs

```diff
@@ -183,6 +458,9 @@ impl BitAnd for EpCtl {
     }
 }
 
+
+
+
```

Comment:
Is this change intentional? 5 newlines in a row is a lot.

Reply:
Unintentional. Fixed.